HelpMyStreetFE/HelpMyStreetFE/ViewComponents/VolunteerListViewComponent.cs

```diff
@@ -53,6 +53,7 @@ public async Task<IViewComponentResult> InvokeAsync(int groupId, CancellationTok
             {
                 return new VolunteerViewModel()
                 {
+                    DateJoined = userInGroup.UserRoleAudit.Where(ur => ur.Action == GroupAction.AddMember && ur.Role == GroupRoles.Member && ur.Success).LastOrDefault()?.DateRequested ?? DateTime.MinValue,
```

Comment:
In the scenario where we can't retrieve a row from the audit table, should we be throwing an error in the scenario rather DateTime.MinValue

Reply:
Previously it did that and broke the whole view!  Having DateTime.MinValue instead renders '01/01/01'.  This is hopefully only an issue in test environments, although we could run a data check in Live.